libkv.go

```diff
@@ -1,3 +1,64 @@
+// Package libkv provides a Go native library to store metadata.
+//
+// The goal of libkv is to abstract common store operations for multiple
```

Comment:
"For multiple" could be confusing here, i.e., can I register in multiple back ends at the same time, or can I pick a single backend from the list of supported backends?